doc/src/find_and_immediate.md

```diff
@@ -166,14 +166,15 @@ MIOpen provides a set of Find modes which are used to accelerate the Find calls.
 
 - `NORMAL`, or `1`: Normal Find: This is the full Find mode call, which will benchmark all the solvers and return a list.
 - `FAST`, or `2`: Fast Find: Checks the [Find-Db](https://rocmsoftwareplatform.github.io/MIOpen/doc/html/finddb.html) for an entry. If there is a Find-Db hit, use that entry. If there is a miss, utilize the Immediate mode fallback. If Start-up times are expected to be faster, but worse GPU performance.
-- `HYBRID`, or `3`, or unset `MIOPEN_FIND_MODE`: Hybrid Find: Checks the [Find-Db](https://rocmsoftwareplatform.github.io/MIOpen/doc/html/finddb.html) for an entry. If there is a Find-Db hit, use that entry. If there is a miss, use the existing Find machinery. Slower start-up times than Fast Find, but no GPU performance drop.
+- `HYBRID`, or `3`: Hybrid Find: Checks the [Find-Db](https://rocmsoftwareplatform.github.io/MIOpen/doc/html/finddb.html) for an entry. If there is a Find-Db hit, use that entry. If there is a miss, use the existing Find machinery. Slower start-up times than Fast Find, but no GPU performance drop.
+- `OPTIMIZED_HYBRID`, or `4`, or unset `MIOPEN_FIND_MODE`: Optimized hybrid Find: Checks the [Find-Db](https://rocmsoftwareplatform.github.io/MIOpen/doc/html/finddb.html) for an entry. If there is a Find-Db hit, use that entry. If there is a miss, use the existing Find machinery with skipping slow-compiling kernels. Faster start-up times than Hybrid Find, may have GPU performance drop.
```

Comment:
I think this would be better to be called `FAST_HYBRID` rather than `OPTIMIZED_HYBRID`. The word "optimize" seems to imply that there would be not GPU performance drop. I also would phrase it as:

> If there is a miss, use the existing Find machinery with a reduced search space. Faster start-up times than Hybrid Find, but may have GPU performance drop.

Reply:
Changed.

Reply:
I would propose "weird hybrid" ;)